module/gnb/api/procedure.go

```diff
@@ -335,6 +335,172 @@ func DeRegistration(ue *RanUeContext) error {
 
 }
 
+// TestGUTIRegistration implements a function registartion using the GUTI after using SUPI reg
+func TestGUTIRegistration() error {
+
+	//registring using UeId -> supi-> suci
+	ueID := "imsi-2089300007487"
+	var recvMsg = make([]byte, 2048)
+
+	ranUeContext, err := Registration(ueID)
+	//check for error
+	if err != nil {
+		logger.GNBLog.Errorln("Error in registration using suci")
+		return err
+	}
+
+	//deregistering using guti
+	err = DeRegistration(ranUeContext)
+	if err != nil {
+		logger.GNBLog.Errorln("Error in degistration")
+		return err
+	}
```

Comment:
Ref to comment before, implement only the second registration not the first registration and deregistration